### ISSUE TITLE
Bug: Pivot Focus outlines

### ISSUE THREAD

**Issue by nico-bellante** [OfficeDev/script-lab#126]
## Environment:

- [ ] production
- [ ] beta
- [x] alpha
- [ ] localhost

## Issue Description:
In IE, the focus outline for each pivot item is missing the right side. Additionally, the contrast ratio is probably not high enough. ( See #42 )

## Screenshots:
![image](https://user-images.githubusercontent.com/35939126/46049058-46a20d00-c0e1-11e8-8c59-ecd6877eb24e.png)

**Comment by gergzk** [comment]
We want to pick up a fix from Fabric for this. Moving the bug to 1.3 since it's not immediately actionable for us.

**Comment by wandyezj** [comment]
IE is no longer supported.
